synapse/cortex.py

```diff
@@ -1046,6 +1099,13 @@ def _logStormQuery(self, text, user):
             async for node in snap.getNodesBy(full, valu, cmpr=cmpr):
                 yield node
 
+    def getCoreInfo(self):
```

Comment:
This is getting very close to duplicating the `stat()` functionality.
Suggestion: Move this data into the stat() results, remove the adminapi helper around the coreApi.stat() information.